docs/source/cpp/compute.rst

```diff
@@ -553,6 +553,25 @@ when a positive ``max_splits`` is given.
   as separator.
 
 
+Slicing
+~~~~~~~
+
+These function transform each sequence of the array to a subsequence, following Python slicing semantics.
+
++--------------------------+------------+-------------------------+-------------------+----------------------------------+---------+
+| Function name            | Arity      | Input types             | Output type       | Options class                    | Notes   |
++==========================+============+=========================+===================+==================================+=========+
+| utf8_slice_codepoints    | Unary      | String-like             | String-like         | :struct:`SliceOptions`         | \(1)    |
++--------------------------+------------+-------------------------+-------------------+----------------------------------+---------+
+
+
+* \(1) Slice string into a substring defined by (`start`, `stop`, `step`) as
+given by `SliceOptions` where `start` is inclusive and `stop` is exclusive and
```

Comment:
Must probably indent these lines to be part of the bullet item (you can click "view file" to get a reST rendering in the Github UI).